app/code/core/Mage/Eav/Model/Config.php

```diff
@@ -28,191 +28,283 @@
-     * Information about preloaded attributes
-     *
-     * @var array
+     * @var Mage_Eav_Model_Entity_Type[]|null
      */
-    protected $_preloadedAttributes              = [];
+    protected $_entityTypeByCode;
 
     /**
-     * Information about entity types with initialized attributes
-     *
-     * @var array
+     * @var Mage_Eav_Model_Entity_Attribute_Abstract[][][]|mixed[][][]|null
      */
-    protected $_initializedAttributes            = [];
+    protected $_entityTypeAttributes;
 
     /**
-     * Attribute codes cache array
-     *
-     * @var array
+     * @var Mage_Eav_Model_Entity_Attribute_Abstract[][][]|null
      */
-    protected $_attributeCodes                   = [];
+    protected $_entityTypeAttributeIdByCode;
 
     /**
-     * Initialized objects
-     *
-     * array ($objectId => $object)
-     *
-     * @var array|null
+     * Attribute set relation information. Structure:
+     * <br/>
+     * [
+     *  int attribute_id => [
+     *      int set_id => [
+     *          int group_id,
+     *          int group_sort,
+     *          int sort,
+     *      ]
+     * ]
+     * @var mixed[][][][]|null
      */
-    protected $_objects;
+    protected $_attributeSetInfo;
 
     /**
-     * References between codes and identifiers
-     *
-     * array (
-     *      'attributes'=> array ($attributeId => $attributeCode),
-     *      'entities'  => array ($entityId => $entityCode)
-     * )
-     *
-     * @var array|null
+     * Special local cache for default attributes to avoid re-hydrating them
+     * @var Mage_Eav_Model_Entity_Attribute_Abstract[][][]|false[][][]
      */
-    protected $_references;
+    protected $_defaultAttributes = [];
 
     /**
      * Cache flag
      *
      * @var bool
      */
-    protected $_isCacheEnabled                    = null;
+    protected $_isCacheEnabled = null;
 
     /**
-     * Array of attributes objects used in collections
-     *
-     * @var array
+     * @var int|false|null
      */
-    protected $_collectionAttributes              = [];
+    protected $_currentStoreId;
+
 
     /**
      * Reset object state
      *
-     * @deprecated
      * @return $this
+     * @deprecated
      */
     public function clear()
     {
-        $this->_entityData            = null;
-        $this->_attributeData         = null;
-        $this->_objects               = null;
-        $this->_references            = null;
-        $this->_preloadedAttributes   = [];
-        $this->_initializedAttributes = [];
+        $this->_storeInitialized = [];
+        $this->_entityTypes = null;
+        $this->_entityTypeByCode = null;
+        $this->_entityTypeAttributes = null;
+        $this->_entityTypeAttributeIdByCode = null;
+        $this->_attributeSetInfo = null;
+        $this->_defaultAttributes = [];
+
+        Mage::app()->cleanCache([self::ENTITIES_CACHE_ID]);
+        // Mage::app()->removeCache(self::ENTITIES_CACHE_ID . '_' . Mage::app()->getStore()->getId());
```

Comment:
```suggestion
```

Reply:
That call was there in a commented state before, didn't want to remove it without cause but will remove it.